dom.bs

```diff
@@ -6485,8 +6518,6 @@ string <var>namespace</var> (default null):</p>
  <var>attr</var>'s <a for=Attr>namespace</a>, <var>attr</var>'s <a for=Attr>local name</a>, and
  <var>element</var>.
 
- <li><p>If <var>oldAttr</var> is <var>attr</var>, return <var>attr</var>.
```

Comment:
This seems like it would change the mutation records for existing applications even when they don't use Trusted Types.

Reply:
This doesn't seem intentional (likely a result of changes that have subsequently been partially reverted), I'll add it back